client/me/help/help-contact/index.jsx

```diff
@@ -286,7 +295,11 @@ module.exports = React.createClass( {
 		//Autofill the subject field since we will be showing it now that operators have went away.
 		this.autofillSubject();
 
-		this.showAvailabilityNotice( IS_UNAVAILABLE );
+		if ( timedOut ) {
```

Comment:
Perhaps the [olark api event](https://github.com/Automattic/wp-calypso/pull/5351/files#diff-bfbe01f0a66b1aa6a6708379328f1d66R46) is calling `onOperatorsAway` with a truthy first argument